featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -266,7 +277,10 @@ def calc_results(time_last, ids, precalculated_features=None, training_window=No
 
         # if all aggregations have been approximated, can calculate all together
         if no_unapproximated_aggs and approximate is not None:
-            grouped = [[datetime.now(), group]]
+            if isinstance(group[target_time].iloc[0], _numeric_types):
+                grouped = [[np.inf, group]]
+            else:
+                grouped = [[datetime.now(), group]]
```

Comment:
should we move this `datetime.now()` outside of `calc_results`. basically, we want every chunk that's part of a single cfm call to use the same time.